app/assets/javascripts/analytics-ga4/ga4-finder-tracker.js

```diff
@@ -83,6 +86,27 @@
             wasFilterRemoved = true
           }
           break
+
+        case 'date': {
+          // The GOV.UK Design System date input consists of three grouped but separate fields (day,
+          // month, year). We want to fire a single event when all three fields are filled in to
+          // avoid firing excessive events.
+          const inputs = Array.from(
+            eventTarget.closest('.govuk-date-input').querySelectorAll('input')
+          )
+          const allInputsSet = inputs.every((input) => input.value)
```

Comment:
```suggestion
          const allInputsSet = inputs.every(input => input.value)
```

Nit: parenthesis aren't required for one argument - I've had a few reviews by frontenders here that are keen that we don't use these parentheses unnecessarily, so sharing the nag